create_table.py

```diff
@@ -1,61 +1,33 @@
 # This sccript creates a totemembeddings table in the postgresql totem database
-import psycopg2
 
-try:
-    # Connect to the default "postgres" database
-    connection = psycopg2.connect(
-        host="localhost",
-        port="5432",
-        database="postgres",
-        user="postgres",
-        password="test"
-    )
-
-    # Create a cursor
-    cursor = connection.cursor()
+import connect_to_db
 
-    # Check if the "totem" database exists
-    cursor.execute("SELECT 1 FROM pg_database WHERE datname='totem'")
-
-    # If it doesn't exist, create it
-    if cursor.fetchone() is None:
-        cursor.execute("COMMIT")
-        cursor.execute("CREATE DATABASE totem")
-
-    # Close cursor and connection
-    cursor.close()
-    connection.close()
-
-    # Totem version
-    connection = psycopg2.connect(
-        host="localhost",
-        port="5432",
-        database="totem",
-        user="postgres",
-        password="test"
-        )
+try:
+    connection = connect_to_db.delete_database()
+    connection = connect_to_db.connect()
 
     print("Connected to the database!")
     # Create a cursor
     cursor = connection.cursor()
 
-    add_extension_query = "CREATE EXTENSION IF NOT EXISTS vector;"  
-    cursor.execute(add_extension_query)
-
-    # Create the table if it doesn't exist
     make_table_query = """
-    CREATE TABLE IF NOT EXISTS totemembeddings (
-        id SERIAL PRIMARY KEY,
-        metadata JSONB UNIQUE,
+    CREATE VIRTUAL TABLE IF NOT EXISTS embeddings USING vec0(embedding float[768]);"""
+
+    make_data_table_query = """
+    CREATE TABLE data (
+        id INTEGER PRIMARY KEY,
+        metadata TEXT,
         page_contents TEXT,
-        embeddings VECTOR);"""
+        embedding_id INTEGER 
+    ) STRICT;"""
```

Comment:
Had to split the embeddings and metadata into different tables due to how the vector extension works.